packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -679,43 +708,17 @@ export class AtomicSearchBox {
         }`}
         role="listbox"
       >
-        {!!this.leftSuggestionElements.length && (
-          <ul
-            part="suggestions suggestions-left"
-            aria-label={this.bindings.i18n.t('query-suggestion-list')}
-            ref={(el) => {
-              this.leftPanelRef = el!;
-            }}
-            class="flex-grow"
-          >
-            {this.leftSuggestionElements.map((suggestion, index) =>
-              this.renderSuggestion(
-                suggestion,
-                index,
-                this.leftSuggestionElements.length - 1,
-                'left'
-              )
-            )}
-          </ul>
+        {this.renderPanel(
```

Comment:
I added a little refactoring here, as all the code is duplicated for these two panels! let me know what you guys think

Reply:
clean!